tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1.md

```diff
@@ -0,0 +1,9 @@
+# Placeholder sayakpaul/deeplabv3-mobilenetv2_dm05-float16/1
+Lightweight deep learning model for semantic image segmentation.
+
+<!-- module-type: image-segmentation -->
+<!-- network-architecture: DeepLab (mobilenetv2_dm05_coco_voc_trainval) -->
+<!-- dataset: PASCAL VOC 2012 -->
+<!-- fine-tunable: false -->
+<!-- language: en -->
```

Comment:
Sorry, this is not documented yet, but the in general the `language` metadata property is usually used for text models to specify the language they were trained or what language their input is expected to be in. 

I think that we can remove the language tag for these set of image models - but please correct me if I'm wrong (I might be missing something regarding how the model is trained or intended to be used).

Reply:
@maringeo there's nothing language-specific in these image models. I just removed it.